data/Drug_target_interaction/meta.yaml

```diff
@@ -0,0 +1,45 @@
+---
+name: Drug-Target Interaction
+description: |-
+    The activity of a small-molecule drug is measured by its binding affinity with the target protein.
+            Given a new target protein, the very first step is to screen a set of potential compounds to find their activity.
+            Traditional method to gauge the affinities are through high-throughput screening wet-lab experiments.
+            However, they are very expensive and are thus restricted by their abilities to search over a large set of candidates
+            Drug-target interaction prediction task aims to predict the interaction activity score in silico given only the accessible compound structural information and protein amino acid sequence.
+targets:
+    - id: binding
+      description: small-molecule protein interaction.
+      units: Kd
+      type: regression
+      names:
+          - Drug-Target Interactionsmall-molecule binding affinity
+          - small-molecule binding
+          - protein-ligand binding
+          - protein-ligandbinding affinity
+          - binding
+identifiers:
+    - id: SMILES
+      type: SMILES
+      description: small-molecule
+    - id: Target
+      type: Other
+      description: Target amino acid sequence
```

Comment:
We would need both to describe an entry. That is the `target` only makes sense if I have both "SMILES" and "Target", right?

Reply:
@kjappelbaum Yes that is correct.  It needs both "SMILES" and "Target" to make sense

Reply:
Also here, we would need some templates to do the sampling correctly (or we update our schema, I'm also open to that if you think this is better)